rust/kcl-lib/src/std/extrude.rs

```diff
@@ -87,6 +100,50 @@ pub async fn extrude(exec_state: &mut ExecState, args: Args) -> Result<KclValue,
 ///
 /// example = extrude(exampleSketch, length = 10)
 /// ```
+///
+/// ```no_run
+/// exampleSketch = startSketchOn('XZ')
```

Comment:
We could use the unquoted planes here too, but I see that the above doc tests do the same. So don't worry about it. We need to go through everything.